doc/source/ray-core/patterns/tree-of-actors.rst

```diff
@@ -0,0 +1,29 @@
+Pattern: Using a supervisor actor to manage a tree of actors
+============================================================
+
+Actor supervision is a pattern that a supervising Ray actor manages a collection of Ray worker actors.
+The supervisor delegates tasks to subordinates and handles their failures.
+This pattern simplifies the driver since it only needs to manage a few supervisors instead of a lot of worker actors
+and doesn't need to deal with failures from worker actors directly.
```

Comment:
```suggestion
This pattern simplifies the driver since it manages only a few supervisors and does not deal with failures from worker actors directly.
```